src/base/bittorrent/filterparserthread.cpp

```diff
@@ -630,7 +510,7 @@ void FilterParserThread::run()
     else if (m_filePath.endsWith(".dat", Qt::CaseInsensitive))
     {
         // eMule DAT format
-        ruleCount = parseDATFilterFile();
+        ruleCount = parseTxtFilterFile<true>();
```

Comment:
Using `bool` parameters causes such confusing code. It's better to use named constants (or enums):
```c++
        ruleCount = parseTxtFilterFile<Format::DAT>();
```